packages/shared/src/components/post/PostActions.tsx

```diff
@@ -166,14 +169,25 @@ export function PostActions({
             Bookmark
           </QuaternaryButton>
           <QuaternaryButton
-            id="share-post-btn"
+            id="copy-post-btn"
             onClick={() => onCopyLinkClick(post)}
             icon={<LinkIcon />}
             responsiveLabelClass={actionsClassName}
-            className="btn-tertiary-cabbage"
+            className="btn-tertiary-cabbage hidden tablet:flex"
           >
             Copy
           </QuaternaryButton>
+          <QuaternaryButton
+            id="share-post-btn"
+            onClick={() =>
+              globalThis?.navigator?.share
+                ? openNativeSharePost(post)
+                : openSharePost({ post })
```

Comment:
Would it be better if we did this comparison in `useSharePost` and export `openSharePost` as either one or the other?

Reply:
Not all cases have to be like that. Only in this implementation.